Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM ubuntu:16.04
```

Comment:
I was using my go-to airdock-based recipe. I've cut over to the standard library python:3.6 now with gosu and tini installed on top.